com.unity.render-pipelines.high-definition/Runtime/Material/Lit/LitPathTracing.hlsl

```diff
@@ -63,13 +63,23 @@ bool CreateMaterialData(PathIntersection pathIntersection, BuiltinData builtinDa
     {
         float NdotV = dot(GetSpecularNormal(mtlData), mtlData.V);
         float Fcoat = F_Schlick(CLEAR_COAT_F0, NdotV);
+
+#if defined(SENSORSDK_SHADERGRAPH) || defined(SENSORSDK_OVERRIDE_REFLECTANCE)
```

Comment:
Given the nature of change here I will suggest to a condition based on some constant buffer value? Avoid to generate a variant

Reply:
ok np, I wasn't sure which one would be prefered.  I guess the branching cost is less significant than the preprocess time of the shaders ?